services/src/messaging/src/main/java/org/openkilda/messaging/info/flow/FlowRerouteResponse.java

```diff
@@ -15,12 +15,31 @@
 
 package org.openkilda.messaging.info.flow;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
 import org.openkilda.messaging.Utils;
+import org.openkilda.messaging.info.InfoData;
 import org.openkilda.messaging.info.event.PathInfoData;
 
-public class FlowRerouteResponse extends FlowPathResponse {
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Value;
+
+/**
+ * Represents a flow reroute northbound response.
+ */
+@JsonSerialize
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Value
+public class FlowRerouteResponse extends InfoData {
```

Comment:
This class has been changed because it inherited not longer compatible DTO. Maybe takes out refactoring of corresponding "conversations" into another pull request?